scripts/split_repo_migration/run.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+ 
+set -e
+ 
+if [ $# -lt 2 ]
+then
+  echo "Usage: $0 <split-repo-name> <target-path>"
+  exit 1
+fi
+ 
+# repo name (e.g. nodejs-asset)
+SPLIT_REPO=$1
+# destination directory (e.g. google-cloud-asset)
+ARTIFACT_NAME=$2
```

Comment:
We were talking about deriving the `$ARTIFACT_NAME` from the split repo, possibly using `jq`. I think that's a good idea (single source of truth, less chance for inconsistencies), though I don't know whether you want to do it here or in `git-migrate-history`.

Reply:
Related: assuming you can derive `$ARTIFACT_NAME`, than that won't need to be `$2`. Instead, make `$2` an optional branch name that you pass to `git-migrate-history` as `$7`

Reply:
Removed artifcat name.